CelesteNet.Shared/CelesteNetClientOptions.cs

```diff
@@ -6,9 +6,12 @@ public class CelesteNetClientOptions {
         public bool AvatarsDisabled = false;
         public ulong ClientID;
         public uint InstanceID;
-        public CelesteNetSupportedClientFeatures SupportedClientFeatures;
+        // Due to how these are sent as packets, this needs to be a ulong.
+        public ulong SupportedClientFeatures = 0;
     }
 
     [Flags]
-    public enum CelesteNetSupportedClientFeatures : ulong {}
+    public enum CelesteNetSupportedClientFeatures : ulong {
+        LocateCommand = 1 >> 0
```

Comment:
Also this is the wrong bit shift operator, although it's technically a no-op x)

1 << 0 = 1
1 << 1 = 2
1 << 2 = 4
etc. for the Flags enum values, right? That was the idea?